tests/e2e/support/objects/app-files/spaces/actions.ts

```diff
@@ -158,3 +164,52 @@ export const changeQuota = async (args: {
 
   await sidebar.close({ page: page })
 }
+
+export interface addMembersArgs {
+  role: string
+  users: User[]
+  page: Page
+}
```

Comment:
can this be reused? same interface is declared in tests/e2e/support/objects/app-files/share/actions.ts too

Reply:
Yes, it can be. Thanks for pointing it out!